specs/merge/beacon-chain.md

```diff
@@ -15,42 +15,41 @@
 - [Constants](#constants)
   - [Execution](#execution)
 - [Configuration](#configuration)
-  - [Transition](#transition)
+  - [Merge](#merge)
 - [Containers](#containers)
   - [Extended containers](#extended-containers)
     - [`BeaconBlockBody`](#beaconblockbody)
     - [`BeaconState`](#beaconstate)
   - [New containers](#new-containers)
     - [`ExecutionPayload`](#executionpayload)
     - [`ExecutionPayloadHeader`](#executionpayloadheader)
-- [Protocols](#protocols)
-  - [`ExecutionEngine`](#executionengine)
-    - [`new_block`](#new_block)
 - [Helper functions](#helper-functions)
-  - [Misc](#misc)
+  - [Predicates](#predicates)
+    - [`is_merge_complete`](#is_merge_complete)
+    - [`is_merge_block`](#is_merge_block)
     - [`is_execution_enabled`](#is_execution_enabled)
-    - [`is_transition_completed`](#is_transition_completed)
-    - [`is_transition_block`](#is_transition_block)
-    - [`compute_time_at_slot`](#compute_time_at_slot)
+  - [Misc](#misc)
+    - [`compute_timestamp_at_slot`](#compute_timestamp_at_slot)
+- [Beacon chain state transition function](#beacon-chain-state-transition-function)
+- [Execution engine](#execution-engine)
+  - [`on_payload`](#on_payload)
   - [Block processing](#block-processing)
-    - [Execution payload processing](#execution-payload-processing)
-      - [`process_execution_payload`](#process_execution_payload)
+  - [Execution payload processing](#execution-payload-processing)
+    - [`process_execution_payload`](#process_execution_payload)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 <!-- /TOC -->
 
 ## Introduction
 
-This is a patch implementing the executable beacon chain proposal. 
-It enshrines transaction execution and validity as a first class citizen at the core of the beacon chain.
+This patch adds transaction execution to the beacon chain as part of the merge.
 
 ## Custom types
 
-We define the following Python custom types for type hinting and readability:
-
 | Name | SSZ equivalent | Description |
 | - | - | - |
-| `OpaqueTransaction` | `ByteList[MAX_BYTES_PER_OPAQUE_TRANSACTION]` | a byte-list containing a single [typed transaction envelope](https://eips.ethereum.org/EIPS/eip-2718#opaque-byte-array-rather-than-an-rlp-array) structured as `TransactionType \|\| TransactionPayload` |
+| `OpaqueTransaction` | `ByteList[MAX_BYTES_PER_OPAQUE_TRANSACTION]` | a [typed transaction envelope](https://eips.ethereum.org/EIPS/eip-2718#opaque-byte-array-rather-than-an-rlp-array) structured as `TransactionType \|\| TransactionPayload` |
+| `Transaction` | **TODO**: define as `Union` type with `OpaqueTransaction` | an execution transaction |
```

Comment:
To define a stub, we should set `Transaction` to `OpaqueTransaction` or `Union[OpaqueTransaction]` for now. I can make pyspec builder obey it. Just ping me once it's decided. :)

Reply:
`Union[OpaqueTransaction]` is a great stub :)

Reply:
done!